package/gluon-alfred/files/lib/gluon/cron/alfred

```diff
@@ -1 +1 @@
-* * * * * /lib/gluon/announce/collect.lua nodeinfo | gzip | alfred -s 158; /lib/gluon/announce/collect.lua statistics | gzip | alfred -s 159; /lib/gluon/announce/collect.lua neighbours | gzip | alfred -s 160
+* * * * * gluon-neighbour-info -d ::1 -p 1001 -t 0 -r nodeinfo | gzip | alfred -s 158; gluon-neighbour-info -d ::1 -p 1001 -t 0 -r statistics | gzip | alfred -s 159; gluon-neighbour-info -d ::1 -p 1001 -t 0 -r neighbours | gzip | alfred -s 160
```

Comment:
That's a good point. We should probably add a single-response feature:

- if a response has been received exit with 0 immediately
- if it runs into the timeout without having received a response, exit with 1

This could be generalized a little as to wait for n responses instead of one. The optional parameter could be called `-c` taking a number as its argument.